configs/pulpcore/3.4.yaml

```diff
@@ -0,0 +1,127 @@
+---
+:project: pulpcore
+:github_org: pulp
+:nightly: false
+:repos: []
+:releases:
+  :3.4:
+    :redmine_version_id: -1
+:mash_scripts:
+  - pulpcore-3.4-el7
+  - pulpcore-3.4-el8
+:tags:
+  - name: pulpcore-3.4-el7
+    based_off: katello-pulpcore-nightly-el7
```

Comment:
Is that just for this first set of tags to be true?

Reply:
Yeah, I want to import the current state in nighly as 3.4 and then update from there